core/Cargo.toml

```diff
@@ -195,8 +195,11 @@ services-vercel-artifacts = []
 services-wasabi = []
 services-webdav = []
 services-webhdfs = []
+
+services-native-hdfs = ["dep:hdfs-native"]
```

Comment:
I'm somewhat hesitant about the names of these services. Maybe use the crate name `services-hdfs-native`?